src/config/cmake/HYPRE_SetupGPUToolkit.cmake

```diff
@@ -6,21 +6,10 @@
 # Enable CXX language
 enable_language(CXX)
 set(CMAKE_CXX_STANDARD_REQUIRED ON)
-if(HYPRE_ENABLE_SYCL)
-  # We enforce the use of Intel's oneAPI DPC++/C++ Compiler
-  if(NOT CMAKE_CXX_COMPILER MATCHES "dpcpp|icpx")
-    message(FATAL_ERROR "SYCL requires DPC++ or Intel C++ compiler")
-  endif()
 
-  # Enforce C++17 at least for SYCL
-  if(NOT DEFINED CMAKE_CXX_STANDARD OR CMAKE_CXX_STANDARD LESS 17)
-    set(CMAKE_CXX_STANDARD 17)
-  endif()
-else()
-  # Enforce C++14 at least for CUDA and HIP
-  if(NOT DEFINED CMAKE_CXX_STANDARD OR CMAKE_CXX_STANDARD LESS 14)
-    set(CMAKE_CXX_STANDARD 14)
-  endif()
+# Enforce C++17 at least
+if(NOT DEFINED CMAKE_CXX_STANDARD OR CMAKE_CXX_STANDARD LESS 17)
```

Comment:
@victorapm , is this overriding the c++ standard set by the user? I'm trying to use cmake on lassen, which does not support c++17 and requires c++11 instead. I tried setting:
-DCMAKE_CXX_STANDARD=11
on my cmake configure line, but it's still using c++17 at compile time.